examples/fpga/artya7/top_artya7.core

```diff
@@ -18,6 +18,11 @@ filesets:
       - data/pins_artya7.xdc
     file_type: xdc
 
+  files_tcl:
+    files:
+      - util/vivado_setup_hooks.tcl: { file_type: tclSource }
+      - util/vivado_hook_write_bitstream_post.tcl: { file_type: user, copyto: vivado_hook_write_bitstream_post.tcl }
```

Comment:
Have these two scripts already been part of the repository before? Otherwise, you have to add them to this very commit, too. Every commit should be atomic. Reverting a single commit shouldn't break the repo.

Reply:
Hi Pirmin, you are right this was a mistake. I tried to fix it now, could you check it please? Thanks

Reply:
This is good now, thanks Chanberk.